option.go

```diff
@@ -53,6 +54,7 @@ var defaultApplicationConfig = config{
 	keyboardLayout:    KeyboardQwertyLayout,
 	windowMode:        WindowModeDefault,
 	windowAlwaysOnTop: false,
+	scrollAmount:      50.0,
```

Comment:
As suggested, let's set the defualt `scroolAmount` to `100.0`

Reply:
Ok, I agree.